test/unit/subnet/ipv4_test.rb

```diff
@@ -226,7 +226,7 @@ def create_a_domain_with_the_subnet
   end
 
   test "should find unused IP on proxy if proxy is set" do
-    subnet = FactoryGirl.create(:subnet_ipv4, :name => 'my_subnet', :network => '192.168.1.0')
+    subnet = FactoryGirl.create(:subnet_ipv4, :ipam_dhcp, :name => 'my_subnet', :network => '192.168.1.0')
```

Comment:
This and other tests down here are really testing the IPAM service class, they should be on a test file of it's own w/o subnet. The `Subnet` test should just test that it calls `suggest_ip`, that's all it ought to be concerned about.